### DISCUSSION TITLE
Latest Update - Watch Screen During Workouts

### DISCUSSION THREAD

**Discussion by gorillaguns** [zepp-health/zeppos-forum#374] [User Feedback]
When the last update was released, the raise to wake feature during workout does not turn off. Mine is off, and the screen still wakes on every wrist raise.

**Comment by MyoungXUE** [comment]
Hello, sorry I just noticed this problem. Does the problem still exist?

Please upgrade Zepp and watch to the latest version first. If the problem still exists, please refer to this method to upload logs and tell me your Zepp ID.


https://docs.zepp.com/docs/guides/faq/upload-log/
